src/Text/Pandoc/Shared.hs

```diff
@@ -630,6 +631,11 @@ compactify'DL items =
              | otherwise           -> items
            _                       -> items
 
+-- | Combine a list of lines by adding hard linebreaks.  This is useful
+--   e.g. for rudimentary support of LineBlock elements in writers.
+combineLines :: [[Inline]] -> [Inline]
+combineLines = concat . map (++ [LineBreak])
```

Comment:
Do we really want a LineBreak at the end of the last line?
Why not

    combineLines = intercalate [LineBreak]

and this is getting so short that maybe we don't even need a separate function?